README.md

```diff
@@ -1,22 +1,35 @@
 # Internet-in-a-Box (IIAB)
 
-Welcome to the Git repository of the Internet-in-a-Box (IIAB) project. This is a community-based project developed and supported by volunteers from around the world. The Internet-in-a-Box (IIAB) is small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection. It provides a local content server of the world’s Free Knowledge.
-
-The older version of this repository is at [github.com/xsce](http://github.com/xsce). We are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
+Welcome to the Git repository for Internet-in-a-Box (IIAB).  This community-based project is enabled by professional volunteers around the world.  Internet-in-a-Box (IIAB) is a small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection.
 
+You are encouraged to customize this local content hotspot or server with the best of the world’s Free Knowledge — to suit your region, your school, your medical establishment, your library — or for your very own family.
 
 ## Installation
 
-Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details about installation of Internet-in-a-Box (IIAB).
+Please read the [Installation](https://github.com/iiab/iiab/wiki/IIAB-Installation) wiki page for details on installing of Internet-in-a-Box (IIAB).
+
+Or, to dive right in, our "1-line installer" is available off of http://download.iiab.io — it's generally best to click on the highest version number.
 
-## Built With
+Please see "What are the best places for community support?" at http://FAQ.IIAB.IO which many answers to help you along the way, in designing the digital knowledge hotspot most suitable for your own community.
 
-* [Ansible](http://www.ansible.com)
+#### Ansible IT Automation
```

Comment:
This should be a level 2 or 3 heading. I would change it back to "Built with Ansible".

Reply:
FWIW level 2 and 3 heading fonts were tested earlier, and both were too big (in my opinion!)